lib/models/work-item.js

```diff
@@ -144,13 +149,19 @@ function WorkItemModelFactory (Model, _, Promise, assert, Constants, configurati
             });
         },
 
-        setFailed: function setFailed(leaseToken, workItems) {
+        setFailed: function setFailed(leaseToken, alertMessage, workItems) {
```

Comment:
Any reason why the `alertMessage` param added in middle of the original params ?  This way you need to make sure to update everywhere the `setFailed` function called.
 
Would it be better to add this as the last param ? so that those code where call the function will still work if you miss to update them.